uds/database/data_record/raw_data_record.py

```
@@ -1,123 +1,62 @@
-"""Definition of RawDataRecord."""
+"""Raw Data Records implementation."""
 
 __all__ = ["RawDataRecord"]
 
-from typing import Optional, Tuple
+from typing import Optional, Sequence
```

Ruff (0.12.2)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)